src/gt4py/next/iterator/transforms/pass_manager.py

```diff
@@ -197,12 +203,12 @@ def apply_common_transforms(
     ir = ScanEtaReduction().visit(ir)
 
     if common_subexpression_elimination:
-        ir = CommonSubexpressionElimination().visit(ir)
+        ir = CommonSubexpressionElimination.apply(ir, offset_provider=offset_provider)  # type: ignore[arg-type]
```

Comment:
Can you quickly explain the #type ignore comment in the code? I'm not able to guess why is needed